### DISCUSSION TITLE
What is minimum required version for Node JS to work with .NET 7 ?

### DISCUSSION THREAD

**Discussion by SHVM-09** [dotnet/aspnetcore#49837] [Q&A] [answered]
Does Node JS version have anything to do with .net 7 ? if so then I should upgrade it or else let it be same?

**Comment by gfoidl** [comment, marked as answer]
> Does Node JS version have anything to do with .net 7 ?

No, they are unrelated.

Was that the question or didn't I understand what you are asking for?
